docs/admin/install/docker.rst

```diff
@@ -1114,6 +1127,14 @@ Example SSL configuration:
 
     .. seealso:: :setting:`django:EMAIL_HOST_PASSWORD`
 
+.. envvar:: WEBLATE_EMAIL_HOST_PASSWORD_FILE
+
+    Path to the file containing the e-mail authentication password.
+
+    .. seealso:: 
```

Comment:
Please remove trailing line:

```suggestion
    .. seealso::
```

(The pre-commit.ci should automatically fix, but it did timeout this time - https://results.pre-commit.ci/run/github/3559019/1619528947.7e1x1TEpRxeBxI7RoxbL4Q)